git-secrets-scan-action/entrypoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+set -e
+
+# Clone and install git-secrets from source
+cd ${HOME}
+git clone https://github.com/awslabs/git-secrets.git && cd git-secrets && make install 
+
+cd ${GITHUB_WORKSPACE}
```

Comment:
As mentioned in https://github.com/aws-robotics/aws-robomaker-github-actions/pull/17#discussion_r502231832, we should add a check here for the value of `${GITHUB_WORKSPACE}`.